### ISSUE TITLE
Suggestion: status-defined image and descriptions

### ISSUE THREAD

**Issue by dhtdht020** [sarisia/actions-status-discord#57]
Hello,
I'm unable to do this myself, but a suggestion I have and would appreciate if implemented, is enabling the option to have a custom image and description per status, for success, cancellation and failure.

I've partially hardcoded this on my own fork to create the following real world example:
![image](https://user-images.githubusercontent.com/18469146/89666843-bb505000-d8e3-11ea-9c54-dcffab7c3c50.png)

**Comment by sarisia** [comment]
Hi, @dhtdht020 , thank you for your suggestion!

I want to keep this action's inputs simple and don't want to add many options,
so I'm not planning to implement this.

However, you can do this by adding some lines to your workflow!
First, place a json file, for example:
```json
{
    "success": "https://img.example.com/success.png",
    "failure": "https://img.example.com/failure.png",
    "cancelled": "https://img.example.com/cancelled.png"
}
```
Then, add a step to your workflow like:
```yaml
steps:
  - uses: actions/checkout@v2
  - id: discord-constants
    run: |
    avatar_url=$(cat discord-constants.json | jq '.${{ job.status }}')
    echo '::set-output name=avatar_url::${avatar_url}'
  - uses: sarisia/actions-status-discord@v1
    with:
      webhook: ${{ secrets.DISCORD_WEBHOOK }}
      status: ${{ job.status }}
      avatar_url: ${{ steps.discord-constants.outputs.avatar_url }}
```
This lets you choose `avatar_url` per `job.status`!
Do same for your `description`.

Anyway, I'm planning to add `noprefix` option which emits status prefixes (like `Success:`, `Failure:` etc.)
from embed title. This may help you customize embed more flexible.

**Comment by dhtdht020** [comment]
That's very useful, thank you so much!
I had no idea you could do something like this with GitHub Actions, the more you know. 😄

**Comment by dhtdht020** [comment]
Additionally, I have a slightly off topic question:
How do I go about having a part of my description lead to an hyperlink?
I've seen this example at note.sarisia.cc:
![image](https://user-images.githubusercontent.com/18469146/89705623-93a2cb80-d967-11ea-8f77-041c9d49607b.png)

**Comment by sarisia** [comment]
it's easy, Discord embed accepts markdown hyperlink syntax!
```yaml
- uses: sarisia/actions-status-discord@v1
        with:
          webhook: ${{ secrets.DISCORD_WEBHOOK_RELEASE }}
          color: 0xff7ac6
          nodetail: true
          job: "New version of `aria-discord-go` is ready!"
          description: "Version `${{ github.event.release.tag_name }}`\nRun update.sh or [Download](https://github.com/aria-music/aria-discord-go/releases)"
```

**Comment by dhtdht020** [comment]
I see now, thank you again!

**Comment by sarisia** [comment]
Just FYI, I've released [v1.6.0](https://github.com/sarisia/actions-status-discord/releases/tag/v1.6.0) which introduces `nocontext` and `noprefix` options!

**Comment by dhtdht020** [comment]
Oh, great!
